.github/workflows/ci.yaml

```diff
@@ -79,3 +79,27 @@ jobs:
     uses: bakdata/ci-templates/.github/workflows/python-poetry-publish-snapshot.yaml@v1.25.0
     secrets:
       pypi-token: ${{ secrets.TEST_PYPI_TOKEN }}
+
+  publish-docs-to-dev:
```

Comment:
should not run from main, it should be a separate job

Reply:
But this is a separate job 🤔

Reply:
Added the job only on PRs